backbone.nativeajax.js

```diff
@@ -11,6 +11,13 @@
   } else if (typeof exports === 'object') { module.exports = factory();
   } else { Backbone.ajax = factory(); }
 }(function() {
+
+  if (!Array.isArray) {
```

Comment:
Yes, please remove the function entirely. IE9 has been EOL'd, and it's trivial to add in a polyfill. Most code these days is targeting ES5+, no need for the extra bloat.